client/js/store/core/VuexApiRoutes.js

```diff
@@ -35,7 +35,7 @@ const api1_0Routes = [
   {
     module: 'role',
     action: 'list',
-    url: '/1.0/role/'
+    url: '/1.0/role'
```

Comment:
Just a question: I see other urls in this file that have the suffix. How do we know in the FE when we have to use it and when we don't?